### PR TITLE
{devel}[GCCcore/12.2.0] line_profiler v4.1.1

### PR DESCRIPTION
**Pull request by NPotvin**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/l/line_profiler/line_profiler-4.1.1-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+
+github_account = 'pyutils'
+source_urls = [GITHUB_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['71c150bf06dd964fc0e5fbc3a5eb31bbee03c535d31c9548999c3e0302dcbb27']
+
+builddependencies = [
+    ('scikit-build', '0.17.2'),
```
Comment (bedroge) [inline easybuild/easyconfigs/l/line_profiler/line_profiler-4.1.1-GCCcore-12.2.0.eb:20]: Thanks for your contribution, and welcome to EasyBuild! :-) 

I compared this easyconfig to an older version and noticed that you lowered the toolchain from `foss` to `GCCcore`; I think this makes sense, as all dependencies depend on `GCCcore` as well, and this library does not seem to be performance-critical or something. However, the CI is unhappy now, as it expects to find a build dependency on `binutils` for tools built with `GCCcore`. So, the following should be an easy fix to pass the CI checks:

```suggestion
    ('binutils', '2.39'),
    ('scikit-build', '0.17.2'),
```

Comment (NPotvin) [reply easybuild/easyconfigs/l/line_profiler/line_profiler-4.1.1-GCCcore-12.2.0.eb:21]: Thanks for your welcome !

I committed your suggestion, it seemed to be compiling fine, so I'm not sure why I need to add binutils.
Is it to ensure the tests pass or is there something more I am missing ?

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.12
See https://gist.github.com/bedroge/583b09035bb2d4a66bd154fba0a61d06 for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen2

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
interactive1 - Linux Rocky Linux 8.8, x86_64, AMD EPYC-Milan Processor (zen2), Python 3.6.8
See https://gist.github.com/bedroge/81eb6e53442574dfbf6ddb7e5d7ac255 for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ jsc-zen2

**Review by bedroge — APPROVED** [review]
Looks good to me.

**Comment by bedroge** [comment]
Going in, thanks @NPotvin!

**Pull request merged**